utils/check_repo.py

```diff
@@ -669,8 +670,11 @@ def find_all_documented_objects():
     "PyTorchBenchmarkArguments",
     "TensorFlowBenchmark",
     "TensorFlowBenchmarkArguments",
+    "MaskFormerSwinBackbone",
     "ResNetBackbone",
     "AutoBackbone",
+    "MaskFormerSwinConfig",
+    "MaskFormerSwinModel",
```

Comment:
Those should be documented in the maskformer page, no reason to wait like for the backbones.

Reply:
I wouldn't document those models, as they might confuse users - it would also add MaskFormerSwin to the main README etc.

Reply:
I've removed them from the main init, and only include them in the "maskformer" module, to make sure all tests can pass. However, `make fixup` complains:

```
python utils/check_inits.py
Traceback (most recent call last):
  File "/Users/nielsrogge/Documents/python_projecten/transformers/utils/check_inits.py", line 299, in <module>
    check_all_inits()
  File "/Users/nielsrogge/Documents/python_projecten/transformers/utils/check_inits.py", line 238, in check_all_inits
    raise ValueError("\n\n".join(failures))
ValueError: Problem in src/transformers/models/maskformer/__init__.py, both halves do not define the same objects.
Differences for torch backend:
  MaskformerSwinModel in TYPE_HINT but not in _import_structure.
  MaskFormerSwinModel in _import_structure but not in TYPE_HINT.
```
This is weird since MaskFormerSwinModel is defined, both in TYPE_HINT and _import_structure.

Reply:
Thanks a mille! 🙏